docs_source_files/content/getting_started_with_webdriver/third_party_drivers_and_plugins.nl.md

```diff
@@ -9,19 +9,18 @@ English to Dutch. Do you speak Dutch? Help us to translate
 it by sending us pull requests!
```

Comment:
@cambiph could you please remove the notice from this file since it is already fully translated? Thanks!

Reply:
Done, sry for the fu**up :)